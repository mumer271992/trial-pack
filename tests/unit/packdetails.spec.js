/* eslint-disable no-undef */
import { mount } from '@vue/test-utils';
import PackDetailsComponent from '../../src/components/PackDetails.vue';

describe('Testing trial pack included items component', () => {
  test('renders correctly', () => {
    const wrapper = mount(PackDetailsComponent);
    expect(wrapper.element).toMatchSnapshot();
  });
});
