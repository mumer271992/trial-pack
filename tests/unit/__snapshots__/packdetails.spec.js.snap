// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing trial pack included items component renders correctly 1`] = `
<section
  id="package_details_section"
>
  <div
    class="container"
  >
    <h2
      class="t-c"
    >
      Contains your test package
    </h2>
     
    <div
      class="c-row"
    >
      <div
        class="c-col-lg-6 c-col-md-6 c-col-sm-6 c-col-xs-12"
      >
        <div
          class="c-row no-wrap"
        >
          <div
            class="c-col-lg-3 c-col-md-3 c-col-sm-4 c-col-xs-4"
          >
            <img
              src="../assets/images/diaper.jpg"
            />
          </div>
           
          <div
            class="c-col-lg-9 c-col-md-9 c-col-sm-8 c-col-xs-8"
          >
            <h4>
              10 LILLYDOO DIAPERS
            </h4>
             
            <ul>
              <li>
                0% perfumes & lotions, 100% LILLYDOO protection
              </li>
               
              <li>
                Extra soft and with an ideal fit
              </li>
            </ul>
          </div>
        </div>
      </div>
       
      <div
        class="c-col-lg-6 c-col-md-6 c-col-sm-6 c-col-xs-12"
      >
        <div
          class="c-row no-wrap"
        >
          <div
            class="c-col-lg-3 c-col-md-3 c-col-sm-4 c-col-xs-4"
          >
            <img
              src="../assets/images/water-wipes-15-small.jpg"
            />
          </div>
           
          <div
            class="c-col-lg-9 c-col-md-9 c-col-sm-8 c-col-xs-8"
          >
            <h4>
              15 WET WIPES WITH 99% WATER
            </h4>
             
            <ul>
              <li>
                0% perfumes & PEGs, 100% biodegradable
              </li>
               
              <li>
                Naturally pure, extra mild, alternative to "water & cotton"
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
